R/utils.R

```diff
@@ -8,15 +8,60 @@
 #' unlink("./my_qcoder_project", recursive=TRUE)
 #' @export
 create_qcoder_project<- function(project_name, sample = FALSE){
+  if(!dir.exists(project_name)){
   dir.create(project_name)
-  dir.create(paste0(project_name, "/documents"))
-  dir.create(paste0(project_name, "/codes"))
-  dir.create(paste0(project_name, "/data_frames"))
-  dir.create(paste0(project_name, "/units"))
-  dir.create(paste0(project_name, "/images"))
+  }
+  else{
+    message("dir: project_name already exists")
+  }
+  if(!dir.exists(file.path(project_name, "/documents"))){
+  dir.create(file.path(project_name, "/documents"))
+  }
+  else{
+    message("dir: project_name, documents already exists")
```

Comment:
Okay this is going to be parallel comments for all of these ... because the whole message is in a string it's not adding the value of  `project_name`.   Instead you need to paste the pieces together like this:
`message(paste0("dir:", project_name, "documents already exists"))`
So that the value of project_name is put in the middle.